xml/System.Diagnostics/Activity.xml

```diff
@@ -172,7 +172,17 @@ This key/value pair is included in the collection returned by the <xref:System.D
         <summary>Adds the specified activity event to the events list.</summary>
         <returns>
           <see langword="this" /> for convenient chaining.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+AddEvent is not intended for adding too many events over a long period because the event objects are stored in the memory. Adding too many events to the same Activity object can have some impact on the app performance.
```

Comment:
```suggestion
`AddEvent` is not intended for adding many events over a long period of time, because the event objects are stored in memory. Adding too many events to the same `Activity` object can impact app performance.
```